src/Compilers/Core/Portable/XmlFileResolver.cs

```diff
@@ -48,7 +48,7 @@ public XmlFileResolver(string? baseDirectory)
         /// If <paramref name="baseFilePath"/> is relative <see cref="BaseDirectory"/> is used as the base path of <paramref name="baseFilePath"/>.
         /// </param>
         /// <returns>Normalized XML document file path or null if not found.</returns>
-        public override string? ResolveReference(string path, string? baseFilePath)
+        public override string? ResolveReference(string? path, string? baseFilePath)
```

Comment:
>string? path [](start = 49, length = 12)

Why is `path` optional? It looks like the uses of `path` support `null` values, but allowing `null` seems at odds with the parameter description where it is a path to the a source file.

Reply:
I'm trying to capture/document the existing behavior of this method.
The test I added does exercise this path. I can adjust the caller instead (don't call if `path` is null), but result is same.

---
In reply to: [443154834](https://github.com/dotnet/roslyn/pull/44266#discussion_r443154834) [](ancestors = 443154834)